pkg/kubectl/cmd/config/delete_context.go

```diff
@@ -77,6 +77,7 @@ func runDeleteContext(out, errOut io.Writer, configAccess clientcmd.ConfigAccess
 	}
 
 	delete(config.Contexts, name)
+	config.CurrentContext = ""
```

Comment:
this will unset any current-context value even though it is not the context to be deleted, right?
why we not unset current-context if only the current-context is context that to be deleted?